test_bot/test_bot.py

```diff
@@ -23,7 +23,7 @@
 from lib.types import CONFIG_DICT_TYPE
 if "pytest" not in sys.modules:
     sys.exit(f"The script {os.path.basename(__file__)} should only be run by pytest.")
-lichess_bot = importlib.import_module("lichess-bot")
+import lib.lichess_bot as lichess_bot
```

Comment:
Is there any reason we don't use `from lib import lichess_bot`?

Reply:
No, just forgot the syntax.